api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
I'm not sure what you're trying to do here. Do you want this version to always match the latest tag on this branch?